services/src/services/update/update.class.js

```diff
@@ -0,0 +1,177 @@
+const FeathersSequelize = require('feathers-sequelize');
+
+/*
+ * This class exist mostly as a wrapper around the feathers-sequelize service
+ * for the Update model.
+ */
+class Update extends FeathersSequelize.Service {
+  constructor(options) {
+    super(options);
+
+    this.app = options.app;
+
+    /*
+     * Undefining some APIs which we don't want/need.
+     */
+    this.update = undefined;
+    this.patch = undefined;
+    this.remove = undefined;
```

Comment:
This will work, but AFAICT this does not seem like the idiomatic way to disable some methods on a given service.
Seems like this is done through https://feathers-plus.github.io/v1/feathers-hooks-common/#disallow

i.e. using hooks, copying for possible future ref in case the link above dies:

```javascript
const { disallow, iff } = require('feathers-hooks-common');
module.exports = { before: {
    // Users can not be created by external access
    create: disallow('external'),
    // A user can not be deleted through the REST provider
    remove: disallow('rest'),
    // disallow calling `update` completely (e.g. to allow only `patch`)
    update: disallow(),
    // disallow the remove hook if the user is not an admin
    remove: iff(context => !context.params.user.isAdmin, disallow())
} };
```

Reply:
feathers-plus is, as far as I can tell, some third party project by [one person](https://github.com/feathers-plus) so I'm not sure I would call it "idiomatic".

We were already guarding this based on `internalOnly`, but the `undefined` completely removes the potential access to these methods and doesn't rely on a hook to do so. Additionally, it ensures that no stubbed out swagger documentation is generated for this